src/assets/translations/en/main.json

```diff
@@ -631,13 +644,6 @@
         "begin_redelegate": "Redelegate",
         "withdraw_delegator_reward": "Claim Reward"
       },
-      "swap": {
-        "deposit": "Swap",
-        "streamingSwap": "Streaming Swap",
-        "out": "Swap",
-        "refund": "Refund",
-        "outbound": "Outbound"
-      },
```

Comment:
We either want to bring this back, or remove the parsing as `swap`:

<img width="342" alt="image" src="https://github.com/shapeshift/web/assets/17035424/4017fbf2-50df-4d42-9905-11188c455f96">
<img width="865" alt="image" src="https://github.com/shapeshift/web/assets/17035424/08a7c97c-b83c-4383-8e78-67e38b704b0b">

Reply:
c03dbe1dcd

Reply:
Yeah I forgot about the thorchain cosmossdk parser, going to bring back until I can sort evm + cosmossdk

Reply:
https://github.com/shapeshift/web/pull/5986/commits/c5b0b23d738d8f992e1f3a279e99bbeab5768e87